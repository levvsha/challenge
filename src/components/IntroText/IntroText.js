import './IntroText.styl';

import React from 'react';

const IntroText = () => (
  <div className="c-intro-text">
    <div className="intro-text-image-wrapper">
      <img
        src="images/mlcc.jpg"
        alt="challenge logo"
        className="intro-text-image"
      />
    </div>
    My solution to the My Little Code Challenge by
    Trustpilot. <a
      target="blank"
      href="https://ponychallenge.trustpilot.com/index.html"
    >
      See details
    </a>.
    <div className="intro-text-image-wrapper">
      <img
        src="images/trustpilot-logo.png"
        alt="trustpilot logo"
        className="intro-text-image"
      />
    </div>
    You can also check my:
    <ul className="my-links-list">
      <li>
        <a
          href="https://stackoverflow.com/users/5806646/mikhail-shabrikov"
          target="_blank"
        >
          Stack Overflow profile
        </a>
      </li>
      <li>
        <a
          href="https://medium.com/@mshabrikov"
          target="_blank"
        >
          articles on Medium
        </a>
      </li>
      <li>
        <a
          href="https://github.com/levvsha"
          target="_blank"
        >
          Github profile
        </a>
      </li>
      <li>
        <a
          href="https://levvsha.github.io/about-me/"
          target="_blank"
        >
          homepage
        </a>
      </li>
    </ul>
  </div>
);

IntroText.displayName = 'IntroText';

// IntroText.propTypes = {};

export default IntroText;
