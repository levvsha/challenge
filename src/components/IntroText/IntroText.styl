.c-intro-text {
  width: 310px

  .intro-text-image {
    height: 100px
  }

  .intro-text-image-wrapper {
    text-align: center
  }

  .my-links-list {
    margin: 15px 0 0 15px
  }
}
