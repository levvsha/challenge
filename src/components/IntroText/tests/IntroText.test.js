import React from 'react';
import renderer from 'react-test-renderer';

import IntroText from '../';

describe('<IntroText />', () => {
  it('IntroText renders correctly', () => {
    const tree = renderer.create(<IntroText />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
