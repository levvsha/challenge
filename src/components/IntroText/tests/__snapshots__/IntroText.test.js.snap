// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IntroText /> IntroText renders correctly 1`] = `
<div
  className="c-intro-text"
>
  <div
    className="intro-text-image-wrapper"
  >
    <img
      alt="challenge logo"
      className="intro-text-image"
      src="images/mlcc.jpg"
    />
  </div>
  My solution to the My Little Code Challenge by Trustpilot. 
  <a
    href="https://ponychallenge.trustpilot.com/index.html"
    target="blank"
  >
    See details
  </a>
  .
  <div
    className="intro-text-image-wrapper"
  >
    <img
      alt="trustpilot logo"
      className="intro-text-image"
      src="images/trustpilot-logo.png"
    />
  </div>
  You can also check my:
  <ul
    className="my-links-list"
  >
    <li>
      <a
        href="https://stackoverflow.com/users/5806646/mikhail-shabrikov"
        target="_blank"
      >
        Stack Overflow profile
      </a>
    </li>
    <li>
      <a
        href="https://medium.com/@mshabrikov"
        target="_blank"
      >
        articles on Medium
      </a>
    </li>
    <li>
      <a
        href="https://github.com/levvsha"
        target="_blank"
      >
        Github profile
      </a>
    </li>
    <li>
      <a
        href="https://levvsha.github.io/about-me/"
        target="_blank"
      >
        homepage
      </a>
    </li>
  </ul>
</div>
`;
